ehrapy/api/io/_utility_io.py

```diff
@@ -1,7 +1,9 @@
 from pathlib import Path, PurePath
 from typing import Union
 
-supported_extensions = {"csv", "tsv", "h5ad"}
+supported_extensions = {"csv", "tsv", "h5ad", "pdf"}
+# file extensions with formats, that can contain multiple tables/data in one file
+multi_data_extensions = {".pdf"}
```

Comment:
1. For consistency `supported_extensions` should then maybe be renamed to single_data_extensions?
2. Using `pdf` above and `.pdf`below. This should be consistent.

Reply:
> 1. For consistency supported_extensions should then maybe be renamed to single_data_extensions?

Don't think so, since this variable still holds references for EVERY valid extension, regardless of whether it's multi data or not.

> Using pdf above and .pdf below. This should be consistent.

Will fix